config/public-api-v1.yml

```diff
@@ -31,31 +31,441 @@ paths:
+          format: markdown
+          description: |
+            Additional information about how the interview process will work for applicants.
+          example: interview_process
+        is_send:
+          type: boolean
+          description: "Does this course have a SEND specialism?"
+          example: true
+        last_published_at:
+          type: string
+          format: date-time
+          description: |
+            Timestamp of when changes to this course's additional information
+            sections was published last.
+          example: "2019-06-13T10:44:31Z"
+        level:
+          type: string
+          description: "The level of pupils this course is designed for."
+          example: secondary
+          enum:
+            - further_education
+            - primary
+            - secondary
+        name:
+          type: string
+          description: "Name of the course."
+          example: "Art and Design"
+        open_for_applications:
+          type: boolean
+          description: "Is the course currently open for applications?"
+          example: true
+        other_requirements:
+          type: string
+          format: markdown
+          description: |
+            Any non-academic qualifications or documents the applicant may need.
+          example: other_requirements
+        personal_qualities:
+          type: string
+          description: |
+            Any skills, motivation and experience the provider is looking in applicants.
+          example: personal_qualities
+        program_type:
+          type: string
+          description: program_type
+          example: scitt_programme
+          enum:
+            - higher_education_programme
+            - scitt_programme
+            - school_direct_training_programme
+            - school_direct_salaried_training_programme
+            - pg_teaching_apprenticeship
+        provider_code:
+          type: string
+          description: "Unique code for the provider of this course."
+          maxLength: 3
+          example: 6CL
+        qualifications:
+          type: string
+          description: |
+            The qualifications as an outcome of the course.
+          example: [ qts ]
+          enum:
+            - [ qts ]
+            - [ pgce ]
+            - [ pgde ]
+            - [ qts, pgce ]
+            - [ qts, pgde ]
+        recruitment_cycle_year:
+          type: string
+          description: The recruitment cycle that this course is available in.
+          example: 2020
+        required_qualifications:
+          type: string
+          format: markdown
+          description: |
+            The minimum academic qualifications needed for this course.
+          example: required_qualifications
+        required_qualifications_english:
+          type: string
+          description: "English GCSE requirements for applicants."
+          example: equivalence_test
+          enum:
+            - ""
```

Comment:
What does blank mean?
where there is `not_required`?

Reply:
I believe that would be `not_set`. It looks like `further_education_course` doesn't require that this be set, so I think that this would be used then.